docs/en/connector-v2/source/InfluxDB.md

```diff
@@ -51,19 +51,20 @@ The query sql used to search data
 select name,age from test
 ```
 
-### fields [string]
+### schema [config]
```

Comment:
Please add change log to this doc and https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md

Reply:
> Please add change log to this doc and https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md

Thanks for your suggestions！I have done this.